kuma/static/styles/components/content.styl

```diff
@@ -276,15 +283,29 @@ p.footnote {
 /* Classes created by the WYSIWYG */
 .twocolumns {
     vendorize(column-count, 2);
+
+    body[contenteditable] & {
+        border: $editor-box-border-style;
+        column-rule: $editor-box-border-style;
+    }
 }
 .threecolumns {
     vendorize(column-count, 3);
+
+    body[contenteditable] & {
+        border: $editor-box-border-style;
+        column-rule: $editor-box-border-style;
+    }
 }
 
 @media $media-query-small-mobile {
    .twocolumns,
    .threecolumns {
         vendorize(column-count, 1);
+        body[contenteditable] & {
```

Comment:
You do not need to define the border styles for the media query, they will be inherited from the desktop styles.